apps/pie-storybook/stories/pie-button.stories.tsx

```diff
@@ -78,6 +79,13 @@ const buttonStoryMeta: ButtonStoryMeta = {
                 summary: false,
             },
         },
+        isResponsive: {
+            description: 'If `true`, enables responsive behaviour.',
```

Comment:
I'd use the same description as you have in the `README` here:

```suggestion
            description: 'If `true`, uses the next larger size on wide viewports',
```